helix-rest/src/test/java/org/apache/helix/rest/client/TestCustomRestClient.java

```diff
@@ -145,6 +150,19 @@ public void testPostRequestFormat() throws IOException {
     Assert.assertEquals(json.get("data").asText(), "{}");
   }
 
+  @Test(expectedExceptions = ConnectTimeoutException.class)
+  public void testPostRequestSmallTimeout() throws IOException {
+    // Set 1 ms to cause timeout for requests
+    System.setProperty(SystemPropertyKeys.HTTP_REQUEST_TIMEOUT, "1");
+    // a popular echo server that echos all the inputs
+    final String echoServer = "http://httpbin.org/post";
+    CustomRestClient _customRestClient = MockCustomRestClientFactory.get();
+    _customRestClient.getInstanceStoppableCheck(echoServer, Collections.emptyMap());
+
+    // Reset the HTTP_REQUEST_TIMEOUT property back to its original value
+    System.setProperty(SystemPropertyKeys.HTTP_REQUEST_TIMEOUT, "60000");
```

Comment:
:"back to its original value" -> where is the original value set?

Reply:
Fixed. Thanks.